common/src/test/java/io/druid/common/guava/GuavaUtilsTest.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.druid.common.guava;
+
+import junit.framework.Assert;
+import org.junit.Test;
+
+import java.util.Objects;
+
+public class GuavaUtilsTest
```

Comment:
Please add the following test cases:

1. `Assert.assertNull(GuavaUtis.tryParseLong("+"))` 
2. `Assert.assertNotEquals(100, GuavaUtils.tryParseLong("+101"))` (or anything showing not equals case)
3. `Assert.assertNull(Longs.tryParse("+100"))` to show prior behavior
4. `Assert.assertNull(GuavaUtils.tryParseLong("++100"))`
4. Negative (`-`) case
5. Null case
5. Empty string case